native/java/org/jpype/manager/TypeManager.java

```diff
@@ -445,6 +445,8 @@ private ClassDescriptor createOrdinaryClass(Class<?> cls, boolean special, boole
 
     // FIXME watch out for anonyous and lambda here.
```

Comment:
I think you can now remove the FIXME (anonymous, or inner classes), right?

Reply:
Would it be possible to handle lambdas in this PR as well?

Reply:
I think that anonymous and  inner classes don't normally make it to this location.   It was only a manually requested class that got here.   I will work to confirm and remove the FIXME in another PR once I have my development machine back.

Reply:
Ah you are still suffering - that's unfortunate. What is going on with your IT-department to take so long to fix a single laptop?

Reply:
The central government agency imposes security requirements on all computer images, and if that central image and all the different levels of group policies has a problem on a particular brand of laptops or such then unfortunately it takes a while to figure out which level of the IT is sending the bad configuration (central. department, or group).   I once lost 4 months on my primary computing server because of a problem with IT installing 4 separate copies of the backup server (without uninstalling the previous versions) and basically leaving my primary machine nonfunctional leaving 2 CPUs with 8 cores and the disk controller pegged 24-7.  

With 20k computers, the thought of one having a problem (or admitting that a central requirement has an issue) is at times challenging.   I am guessing that many people just don't notice their computer is dog slow and just put up with it (hence the perception everything is working fine).  I don't blame the local team as 90% of the time it isn't coming from them.  For example, the laptop came supplied with drivers and such from Dell, it was then reimaged to drivers and windows from 12 months ago.   Central are still evaluating the windows from 9 months ago, and that should be installed next month, at which time they will start evaluating the next version.  Yes, they backport all security fixes so we are bleeding edge on that front, but the actual image is always very out of date.  Real fun starts when their backport patch fails and the whole site as buggy or broken images for a month.   I can't imagine what life is like in our IT department, but  I would wager it is demoralizing at best.